packages/node_modules/@webex/plugin-messages/test/integration/spec/messages.js

```diff
@@ -23,40 +23,71 @@ describe('plugin-messages', function () {
   this.timeout(60000);
 
   let webex;
+  let webexEU;
   let actor;
+  let actorEU;
+
+
+  before(() => Promise.all([
+    testUsers.create({count: 1}),
+    testUsers.create({count: 1, config: {orgId: process.env.EU_PRIMARY_ORG_ID}})
+  ])
+    .then(([user, usersEU]) => {
+      [actor] = user;
+      [actorEU] = usersEU;
+
+      webex = new WebexCore({credentials: actor.token});
+      webexEU = new WebexCore({credentials: actorEU.token});
 
-  before(() => testUsers.create({count: 1})
-    .then(([user]) => {
-      webex = new WebexCore({credentials: user.token});
       webex.people.get('me')
         .then((person) => {
           actor = person;
         });
+
+      webexEU.people.get('me')
+        .then((person) => {
+          actorEU = person;
+        });
     }));
 
   describe('#messages', () => {
     let room;
+    let roomEU;
 
-    before(() => webex.rooms.create({title: 'Webex Teams Test Room'})
-      .then((r) => {
+    before(() => Promise.all([
+      webex.rooms.create({title: 'Webex Teams Test Room'}),
+      webexEU.rooms.create({title: 'Webex Teams Test Room for EU'})
+    ])
+      .then(([r, rEU]) => {
         room = r;
+        roomEU = rEU;
         const text = 'First Message';
 
-        return webex.messages.create({
+        webex.messages.create({
           roomId: room.id,
           text
         })
           .then((message) => {
             validateMessage(message, text);
           });
+
+        webexEU.messages.create({
+          roomId: roomEU.id,
+          text
+        })
+          .then((message) => {
+            validateMessage(message, text);
+          });
       }));
 
+    // eslint-disable-next-line consistent-return
     after(() => {
-      if (!room) {
-        return Promise.resolve();
-      }
+      webex.rooms.remove(room)
```

Comment:
Let's keep this async and return a promise all:

```suggestion
      return Promise.all([
        webex.rooms.remove(room),
        webexEU.rooms.remove(roomEU)
      ]);
```

Reply:
Yeah, that's a good idea! Just updated, thanks @adamweeks